src/tickit/core/management/event_router.py

```diff
@@ -169,7 +171,11 @@ def components(self) -> Set[ComponentID]:
         Returns:
             Set[ComponentID]: A set of all components in the wiring.
         """
-        return set.union(self.input_components, self.output_components)
+        return set.union(
+            self.input_components,
+            self.output_components,
+            self.isolated_components,
```

Comment:
re: discussion, looks easier to parse than just returning {key for key, _ in self.wiring.items()}, so happy to leave